ecal2val/proto_struct/vss_data.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package proto_struct;
+
+message VssData
+{
+  string description = 1;
+  int32  data_int    = 2;
```

Comment:
Couldn't you leverage the structure of https://github.com/eclipse/kuksa.val/blob/master/proto/kuksa/val/v1/types.proto so for example use Datapoint etc. directly and then you not need to handle that much for float/bool/string etc.